pkg/testcommon/testcommon.go

```diff
@@ -320,17 +327,22 @@ func GetCachedArchive(siteName string, prefixString string, internalExtractionPa
 	if err != nil {
 		return extractPath, "", fmt.Errorf("failed to remove %s: %v", extractPath, err)
 	}
+
 	if filepath.Ext(archiveFullPath) == ".zip" {
 		err = archive.Unzip(archiveFullPath, extractPath, internalExtractionPath)
 	} else {
 		err = archive.Untar(archiveFullPath, extractPath, internalExtractionPath)
 	}
+
 	if err != nil {
 		_ = fileutil.PurgeDirectory(extractPath)
 		_ = os.RemoveAll(extractPath)
 		_ = os.RemoveAll(archiveFullPath)
 		return extractPath, archiveFullPath, fmt.Errorf("archive extraction of %s failed err=%v", archiveFullPath, err)
 	}
+
+	output.UserOut.Printf("Extracted %s into %s", archiveFullPath, extractPath)
```

Comment:
Judging from my manual test this seems not to have the right info.

Reply:
No, this is fine but will change anyway again because of the requested file name change.